src/lib/deploy-actions.js

```diff
@@ -25,7 +25,15 @@ module.exports = async (config, isLocal = false, log = () => {}) => {
   utils.runScript(config.hooks['pre-app-deploy'])
   const script = await utils.runScript(config.hooks['deploy-actions'])
   if (!script) {
-    const entities = await deployActions(config, { isLocalDev: isLocal }, log)
+    const deployConfig = {
+      isLocalDev: isLocal
+    }
+    if (config.filterByBuiltActions) {
+      deployConfig.filterEntities = {}
```

Comment:
please don't add fields to the config object, it has a specific format as defined by the config-loader, I don't think we should mess up with this spec. Any way to pass this as an additional argument ?